tests/Functional/Anonymizer/Core/FloatAnonymizerTest.php

```diff
@@ -44,6 +45,8 @@ protected function createTestData(): void
 
     public function testAnonymize(): void
     {
+        $this->skipIfDatabase(AbstractBridge::SERVER_SQLITE, 'SQLite RANDOM() return an int, not a float, requires specific code.');
```

Comment:
I don't understand the problem here, this anonymizer use ::getRandomExpression which return an integer, then the expresion is 
```
$expr->raw(
                    'FLOOR(? * (? - ? + 1) + ?) / ?',
                    [
                        $this->getRandomExpression(),
                        $expr->cast($max * $precision, 'int'),
                        $min * $precision,
                        $min * $precision,
                        $precision
                    ]
                )
```

This should work ?

Reply:
Hum, with randomInt() yes, it might do the trick !